pkgs/by-name/le/less/package.nix

```
@@ -32,11 +32,15 @@ stdenv.mkDerivation (finalAttrs: {
     "man"
   ];
 
-  configureFlags = [
-    "--sysconfdir=/etc" # Look for 'sysless' in /etc
-    (lib.withFeatureAs true "regex" "pcre2")
-    (lib.withFeature withSecure "secure")
-  ];
+  configureFlags =
+    [
+      "--sysconfdir=/etc" # Look for 'sysless' in /etc
+      (lib.withFeatureAs true "regex" "pcre2")
+    ]
+    # Normally we should add `lib.withFeature withSecure "secure"` to configureFlags,
+    # but `--without-secure` does not work as expected in less-668, it enables secure
+    # mode instead, so only add `--with-secure` if enabled.
+    ++ lib.optional withSecure "--with-secure";
```

```suggestion
    ++ lib.optionals withSecure [ (lib.withFeature true "secure") ];
```

I don't like the optional.